min_stack.py

```diff
@@ -0,0 +1,71 @@
+import sys
+
+class MinStack:
+
+    def __init__(self):
+        """
+        initialize your data structure here.
+        """
+        self.stack = []
+        self.min=sys.maxsize
+
+    def push(self, x: int) -> None:
+        """
+        if incoming is less or equal to min than push(min) -> min=incoming -> push(incoming)
+        """
+        if x >= self.min:
```

Comment:
This should be strictly greater (>) not >=

Reply:
understood. Thanks

Reply:
desgin_hashmap shows error while submitting at Leetcode. It throws error when a large number of operations got performed.